.gitignore

```diff
@@ -47,3 +47,5 @@ yarn.lock
 *.iml
 *.ipr
 *.iws
+
+dist
```

Comment:
I could not get the HTMLWebpackPlugin to not generate the files in the `dist` folder, so I just added it to the `.gitignore`.